hack/jenkins/e2e.sh

```diff
@@ -612,26 +612,76 @@ case ${JOB_NAME} in
     : ${E2E_UP:="false"}
     : ${E2E_TEST:="true"}
     : ${E2E_DOWN:="false"}
-    : ${GINKGO_TEST_ARGS:="--ginkgo.focus=Skipped.*Cluster\supgrade.*upgrade-cluster"}
-    NUM_MINIONS=3
+    : ${GINKGO_TEST_ARGS:="--ginkgo.focus=Skipped.*Cluster\supgrade.*upgrade-master"}
+    : ${NUM_MINIONS:=3}
```

Comment:
default is 3 https://github.com/kubernetes/kubernetes/blob/master/cluster/gce/config-test.sh#L23. I'm being explicit here to work around some weird behavior I was seeing where some steps were not picking up the test config. I admit that I have not fully investigated why but the reason was not obvious.